src/main/java/create_task_use_case/CreateTaskDsOutputData.java

```diff
@@ -0,0 +1,44 @@
+/**
+ * Create Task DsInputData Class.
+ * Author: Fardin Faruk
+ * Modified By: N/A
+ * Created: Nov 19, 2022
+ * Last Modified: N/A
+ */
+package create_task_use_case;
+
+public class CreateTaskDsOutputData {
+    private String title;
+    private boolean completed;
+    private int dayID;
+
+    public CreateTaskDsOutputData(String title, boolean completed, int dayID){
```

Comment:
This is only used as the input, not the output, for Data Access classes. Should be renamed to CreateTaskDsInputData.

Reply:
Done.